ceph/nvmegw_cli/gateway.py

```diff
@@ -21,3 +21,6 @@ def info(self, **kwargs):
 
     def version(self, **kwargs):
         return self.run_nvme_cli("version", **kwargs)
+
+    def set_log_level(self, **kwargs):
```

Comment:
Log level is already present in package.

Reply:
I was unable to find the log level package for gateway, whereas its present for spdk_log_level. Please let me know if I am wrong.

Reply:
https://github.com/red-hat-storage/cephci/blob/master/ceph/nvmegw_cli/log_level.py

Reply:
This is for spdk_log_level
I am trying to set loglevel at GW level like `podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.3.1-1  --server-address 10.0.66.56 gw set_log_level --level ERROR`

Reply:
```
root@ceph-sunilkumar-01-y0ty7q-node6 ~]# podman run --quiet --rm  -v /etc/mtls/server.crt:/root/server.crt:z  -v /etc/mtls/client.crt:/root/client.crt:z  -v /etc/mtls/client.key:/root/client.key:z  cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.3.1-1 --client-key /root/client.key --client-cert /root/client.crt --server-cert /root/server.crt --server-address 10.0.64.167 --server-port 5500 spdk_log_level --help
usage: python3 -m control.cli spdk_log_level [-h] {get,set,disable} ...

SPDK nvmf log level commands

optional arguments:
  -h, --help         show this help message and exit

Action:
  {get,set,disable}
    get              Get SPDK log levels and nvmf log flags
    set              Set SPDK log levels and nvmf log flags
    disable          Disable SPDK nvmf log flags
 ```

Reply:
I am talking about this
```[root@dhcp46-241 ~]# podman run --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.3.1-1 --server-address 10.70.46.241 --server-port 5500 gw --help
usage: python3 -m control.cli gw [-h]
                                 {version,info,get_log_level,set_log_level}
                                 ...

Gateway commands

optional arguments:
  -h, --help            show this help message and exit

Action:
  {version,info,get_log_level,set_log_level}
    version             Display gateway's version
    info                Display gateway's information
    get_log_level       Get gateway's log level
    set_log_level       Set gateway's log level```